pkg/collector/corechecks/servicediscovery/impl_linux_test.go

```diff
@@ -59,6 +59,15 @@ var (
 			Starttime: procLaunchedClicks,
 		},
 	}
+	procPythonService = testProc{
```

Comment:
let's add tests that are not covered by the current language detection algorithm

Reply:
The other existing tests cover the case where the language detection algorithm fails.

Reply:
The other existing tests cover the case where the language detection algorithm fails.